src/pages/Campaign/index.tsx

```diff
@@ -219,9 +219,35 @@ export default function Campaign() {
   useInterval(
     () => {
       if (selectedCampaign && selectedCampaign.status === 'Upcoming' && selectedCampaign.startTime < now + 1000) {
+        dispatch(
+          setCampaignData({
+            campaigns: campaigns.map(campaign => {
+              if (campaign.id === selectedCampaign.id) {
+                return {
+                  ...campaign,
+                  status: 'Ongoing',
+                }
+              }
+              return campaign
+            }),
+          }),
+        )
         dispatch(setSelectedCampaign({ campaign: { ...selectedCampaign, status: 'Ongoing' } }))
       }
       if (selectedCampaign && selectedCampaign.status === 'Ongoing' && selectedCampaign.endTime < now + 1000) {
+        dispatch(
+          setCampaignData({
+            campaigns: campaigns.map(campaign => {
+              if (campaign.id === selectedCampaign.id) {
+                return {
+                  ...campaign,
+                  status: 'Ended',
```

Comment:
I think u can write a constant for that status, I saw many statuses in your code.

Reply:
Sure, will refactor soon in other PR. This PR is urgent for release.